R/create.ccf.summary.heatmap.R

```diff
@@ -88,27 +112,56 @@ create.ccf.summary.heatmap <- function(
         list(
             legend = list(
                 title = 'CCF',
-                labels = c(min(arr), max(arr)),
+                labels = c(min(arr), rep('', legend.size), max(arr)),
                 colours = c('white', 'blue'),
                 border = 'black',
-                continuous = TRUE
+                continuous = TRUE,
+                cex = legend.label.cex
                 )
             ),
         size = legend.size,
         title.cex = legend.title.cex,
         label.cex = legend.label.cex
         );
 
+    if (!is.null(clone.colours)) {
+        clone.cov <- BoutrosLab.plotting.general::create.heatmap(
+            x = t(clone.colours[rownames(arr)]),
+            xlab.label = 'Clone ID',
+            xlab.cex = subplot.xlab.cex,
+            xaxis.lab = rownames(arr),
+            xaxis.cex = subplot.xaxis.cex,
+            xaxis.fontface = subplot.xaxis.fontface,
+            xaxis.rot = hm.xaxis.rot,
+            input.colours = TRUE,
+            clustering.method = 'none',
+            grid.col = FALSE,
+            print.colour.key = FALSE,
+            yaxis.tck = 0
+            );
+        plot.list <- list(clone.bar, hm, sample.bar, clone.cov);
+        layout.skip <- c(FALSE, TRUE, FALSE, FALSE, FALSE, TRUE);
+        layout.height <- 3;
+        plot.objects.heights <- c(0.3, 0.9, 0.2);
```

Comment:
Can we avoid the magic numbers for `plot.objects.heights`? Just some named variables might help for clarity.